### ISSUE TITLE
cron expression with time range and step value throws an exception

### ISSUE THREAD

**Issue by satagrawal** [jamesmh/coravel#172]
**Describe the bug**
I am using a cron expression for scheduling.  The cron expression is 0 17-23/2 * * 1-5.  So run for every two hours starting at 5pm and run from monday thru friday.  When the cron expression is evaluated, it throws an exception $17-23/2 is malformed.

**Affected Coravel Feature**
Scheduling

**Expected behaviour**
I would have expected it to run as intended

**Comment by jamesmh** [comment]
Yes, I don't think that case is handled currently.

For now, you could schedule the same invocable/job multiple times:

```csharp
job.DailyAtHour(18).Weekday();

job2.DailyAtHour(20).Weekday();

job3.DailyAtHour(22).Weekday();
```

**Comment by satagrawal** [comment]
Thanks for your response.  Any future plans to get that working?

**Comment by jamesmh** [comment]
I'll look at top-priorities when I get around to it 😉 No guarantees right now lol

**Comment by satagrawal** [comment]
I might be able to take a look and add that functionality.

**Comment by jamesmh** [comment]
Sure, the fix will be here: https://github.com/jamesmh/coravel/blob/master/Src/Coravel/Scheduling/Schedule/Cron/CronExpressionPart.cs#L49

You can add some test cases for this here: https://github.com/jamesmh/coravel/blob/master/Src/UnitTests/CoravelUnitTests/Scheduling/CronTests/CronExpressionTests.cs

**Comment by jamesmh** [comment]
This is available in 4.0.4. Please let me know if this doesn't work / does work for you. Will close the issue for now. 

Thanks!
